### PR TITLE
add Github Issue Template

### PR DESCRIPTION
**Pull request by chrisbra**
This PR is about to improve the documentation regarding reporting issues. It contains of 2 parts basically:

```
More and more people are reporting user questions here, so therefore improve the Contributing guidelines, so people know how to write good bug reports.
Create an ISSUE_TEMPLATE.md file, that will be used as template, when a new issue is created. This works as a basic questionnaire to ask for relevent information from the reporter and also make sure, people read the contribution guidelines. (since this is a PR from my private repository, you can see how this works here)
```

Comments on how to improve the situation are welcome.

### CODE REVIEW

**Comment by justinmk** [comment]
Looks like YCM's issue template, which is designed to scare away questions. Too bureaucratic, IMO: too long, too wordy, too many questions.

**Comment by chrisbra** [comment]
;)
I don't use YCM and did not look at their issue tracker, but rather let me inspire from git for windows. And yes, part of the motiviation is to have user report better (and do not ask user questions).
